deep_quoridor/src/arena.py

```diff
@@ -128,8 +128,8 @@ def play_games(self, players: list[str], times: int):
         for i in range(len(players)):
             for j in range(i + 1, len(players)):
                 for t in range(times):
-                    agent_i = Agent.create(players[i])
-                    agent_j = Agent.create(players[j])
+                    agent_i = Agent.create(players[i], board_size=self.board_size)
+                    agent_j = Agent.create(players[j], board_size=self.board_size)
```

Comment:
Not for this PR, but I think we should be passing here either the Arena or a new data class ArenaInfo or something.  E.g. we'll probably want the number of walls in the future

Reply:
Yes. Note that I added `**kwargs` in the init of the agents, so that should allow passing anything needed.